logstash-event.gemspec

```diff
@@ -25,4 +26,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec"
```

Comment:
We are missing a few dependencies here?

require "cabin"
require "logstash/namespace"
require "logstash/util/accessors"
require "logstash/timestamp"